.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -26,15 +26,17 @@ A clear and concise description of what you expected to happen.
 If applicable, add screenshots to help explain your problem.
 
 **Setup (please complete the following information):**
- - OS: [e.g. iOS]
- - IDE : [e.g. IntelliJ]
- - Version of the Library used
+
+- OS: [e.g. iOS]
+- IDE : [e.g. IntelliJ]
+- Version of the Library used
 
 **Additional context**
 Add any other context about the problem here.
 
 **Information Checklist**
 Kindly make sure that you have added all the following information above and checkoff the required fields otherwise we will treat the issuer as an incomplete report
```

Comment:
Consider changing it to "please" - https://ell.stackexchange.com/questions/51618/difference-between-kindly-and-please